sdk/communication/Azure.Communication.Common/src/Identity/CommunicationTokenCredential.cs

```diff
@@ -9,33 +9,33 @@
 namespace Azure.Communication.Identity
 {
     /// <summary>
-    /// The Azure Communication Services User token credential.
+    /// The Azure Communication Services Token Credential.
     /// </summary>
-    public sealed class CommunicationUserCredential : IDisposable
+    public sealed class CommunicationTokenCredential : IDisposable
```

Comment:
If I remember correctly we talked about getting rid of the Azure.Communication.Identity namespace and move the credential up into Azure.Communication.

@KrzysztofCwalina Does that sound right?

Reply:
I will address this in my next PR!